wled00/wled02_xml.ino

```diff
@@ -249,7 +249,7 @@ void getSettingsJS(byte subPage, char* dest)
   if (subPage == 4)
   {
     sappend('c',"BT",buttonEnabled);
-    sappend('c',"IR",irEnabled);
+    sappend('v',"IR",irEnabled);
```

Comment:
this changes the "ON/OFF" checkbox to a value entry field (0 to 4) with 0=disabled, 1=1st remote, 2=2nd remote etc. /// 4 different IR remotes in total.